src/rpc/rawtransaction.cpp

```diff
@@ -910,13 +906,9 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
     CTransactionRef tx(MakeTransactionRef(std::move(mtx)));
     const uint256& tx_hash = tx->GetHash();
 
-    CFeeRate max_raw_tx_fee_rate = DEFAULT_MAX_RAW_TX_FEE_RATE;
-    // TODO: temporary migration code for old clients. Remove in v0.20
```

Comment:
Does this breaking change to the RPC API need to go through an `-deprecatedrpc` cycle? See https://github.com/bitcoin/bitcoin/blob/master/doc/JSON-RPC-interface.md#versioning

Reply:
Maybe not because it is already broken :) That is - even in `master` now a `bool` argument is not accepted.

Reply:
lolwut

Could add a test to this pull to check that it still fails?